### ISSUE TITLE
[Bug] Having trouble with installation

### ISSUE THREAD

**Issue by summersab** [salgum1114/react-design-editor#149]
First, let me be clear that I'm a total Node n00b and a PHP guy. When I try to run the project with `npm start`, the following happens:
```
> react-design-editor@0.0.17 start /home/user/designer/react-design-editor
> npm install && npm run start:dev

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.11 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.11: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 14994 packages in 10.357s

30 packages are looking for funding
  run `npm fund` for details

found 4 moderate severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details

> react-design-editor@0.0.17 start:dev /home/user/designer/react-design-editor
> webpack-dev-server --config webpack.dev.js --inline

ℹ ｢wds｣: Project is running at http://localhost:8080/
ℹ ｢wds｣: webpack output is served from /
ℹ ｢wds｣: Content not from webpack is served from /home/user/designer/react-design-editor/public
ℹ ｢wds｣: 404s will fallback to /index.html
```

It gets to here, and then I try to hit the page with a browser. That's when this happens:
```
ℹ ｢wdm｣: wait until bundle finished: /
✖ ｢wdm｣: Hash: 68372c01b54990f41faa
Version: webpack 4.41.5
Time: 32784ms
Built at: 01/07/2020 7:05:06 PM
 129 assets
Entrypoint bundle = vendor.68372c01b54990f4.js bundle.68372c01b54990f4.js
[0] multi (webpack)-dev-server/client?http://localhost:8080 (webpack)/hot/dev-server.js @babel/polyfill react-hot-loader/patch webpack-dev-server/client?http://localhost:8080 webpack/hot/only-dev-server ./src/index.js 100 bytes {bundle} [built]
[./node_modules/@babel/polyfill/lib/index.js] 686 bytes {vendor} [built]
[./node_modules/@babel/polyfill/lib/noConflict.js] 567 bytes {vendor} [built]
[./node_modules/antd/lib/locale-provider/en_US.js] 339 bytes {vendor} [built]
[./node_modules/antd/lib/locale-provider/index.js] 5.01 KiB {vendor} [built]
[./node_modules/antd/lib/locale-provider/ko_KR.js] 1.35 KiB {vendor} [built]
[./node_modules/antd/lib/locale-provider/style/index.js] 39 bytes {vendor} [built]
[./node_modules/core-js/library/fn/global.js] 87 bytes {vendor} [built]
[./node_modules/react-dom/index.js] 1.33 KiB {vendor} [built]
[./node_modules/react-hot-loader/dist/react-hot-loader.development.js] 97.1 KiB {vendor} [built]
[./node_modules/react-hot-loader/patch.js] 229 bytes {vendor} [built]
[./node_modules/webpack-dev-server/client/index.js?http://localhost:8080] (webpack)-dev-server/client?http://localhost:8080 4.29 KiB {vendor} [built]
[./node_modules/webpack/hot/dev-server.js] (webpack)/hot/dev-server.js 1.59 KiB {vendor} [built]
[./node_modules/webpack/hot/only-dev-server.js] (webpack)/hot/only-dev-server.js 2.52 KiB {vendor} [built]
[./src/index.js] 1.99 KiB {bundle} [built]
    + 2175 hidden modules

ERROR in ./src/components/canvas/objects/index.ts
Module not found: Error: Can't resolve './Iframe' in '/home/user/designer/react-design-editor/src/components/canvas/objects'
 @ ./src/components/canvas/objects/index.ts 9:0-45 9:0-45
 @ ./src/components/workflow/link/index.js
 @ ./src/components/workflow/WorkflowEditor.js
 @ ./src/containers/App.js
 @ ./src/index.js
 @ multi @babel/polyfill react-hot-loader/patch webpack-dev-server/client?http://localhost:8080 webpack/hot/only-dev-server ./src/index.js

ERROR in ./src/components/imagemap/properties/PropertyDefinition.js
Module not found: Error: Can't resolve './IframeProperty' in '/home/user/designer/react-design-editor/src/components/imagemap/properties'
 @ ./src/components/imagemap/properties/PropertyDefinition.js 20:0-46 397:17-31
 @ ./src/components/imagemap/properties/NodeProperties.js
 @ ./src/components/imagemap/ImageMapConfigurations.js
 @ ./src/components/imagemap/ImageMapEditor.js
 @ ./src/containers/App.js
 @ ./src/index.js
 @ multi @babel/polyfill react-hot-loader/patch webpack-dev-server/client?http://localhost:8080 webpack/hot/only-dev-server ./src/index.js
Child html-webpack-plugin for "index.html":
     1 asset
    Entrypoint undefined = index.html
    [./node_modules/html-webpack-plugin/lib/loader.js!./node_modules/html-webpack-plugin/default_index.ejs] 376 bytes {0} [built]
    [./node_modules/lodash/lodash.js] 528 KiB {0} [built]
    [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 472 bytes {0} [built]
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 497 bytes {0} [built]
ℹ ｢wdm｣: Failed to compile.
```

What's going on? I'm running a fairly-vanilla Debian build with the latest version of Node and npm installed, but I just don't know enough about Node to figure this out.

Thanks!

**Comment by salgum1114** [comment]
@summersab 

It fixes bug in #145

**Comment by summersab** [comment]
Me love you long time. Worked perfectly!
```
mv ./src/components/imagemap/properties/IFrameProperty.js ./src/components/imagemap/properties/IframeProperty.js
mv ./src/components/canvas/objects/IFrame.ts ./src/components/canvas/objects/Iframe.ts
```
